packages/jellyfish-api-core/__tests__/category/loan/listVaults.test.ts

```diff
@@ -580,5 +606,5 @@ describe('Loan listVaults with options and pagination', () => {
     const vaults = await testing.rpc.loan.listVaults()
     const theLiqVault = vaults.find(vault => vault.vaultId === vaultId)!
     expect(theLiqVault.state).toStrictEqual(VaultState.MAY_LIQUIDATE)
-  })
+  }, 480000)
```

Comment:
```suggestion
  })
```

Reply:
added this one on purpose. as it kept having timeout issue on my local and github action

Reply:
hmm.. maybe the timeout added is special for this
the log was for reference
```sh
console.time
    frozen: 3928 ms

      at packages/jellyfish-api-core/__tests__/category/loan/listVaults.test.ts:581:13
          at runMicrotasks (<anonymous>)

  console.time
    back to active: 3957 ms

      at packages/jellyfish-api-core/__tests__/category/loan/listVaults.test.ts:592:13
          at runMicrotasks (<anonymous>)

  console.time
    googl: 8147 ms

      at packages/jellyfish-api-core/__tests__/category/loan/listVaults.test.ts:602:13
          at runMicrotasks (<anonymous>)

  console.time
    end auction: 23795 ms
  ```

Reply:
feel the `timeout` is still not an appropriate solution.. added a minor fix for it